playing-card-detector/newProject/python/serverClient.py

```diff
@@ -0,0 +1,132 @@
+import socket
+import sys
+import signal
+import cv2
+import numpy as np
```

Comment:
What is the difference between serverClient.py and cvDev.py? Could some code be reused and included/imported so that we don't have two files with almost identical functionality?

Reply:
The difference is stated in the comment on the top, that cvDev is for trying out cv things without having to worry about a server, and it's true that code duplication is bad, but I don't want to rely on outside files for the serverClient. cvDev will likely be deleted a bit later.